### PR TITLE
Letter-spacing-ios-device

### PR DESCRIPTION
**Pull request by cp-priyanka-g**

### CODE REVIEW

File: vue-frontend/src/components/home-new/LandingSection.vue [hunk 1]
```diff
@@ -380,4 +380,10 @@ export default {
     line-height: 5.875rem;
   }
 }
+
+@supports (-webkit-touch-callout: none) {
+  .v2-header-3-text {
+    letter-spacing: -0px;
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/LandingSection.vue:386]: What is the different between 0px and -0px ?? Is it valid css property value ??

File: vue-frontend/src/components/home-new/BlogSection.vue [hunk 2]
```diff
@@ -270,4 +270,9 @@ export default {
     margin-top: -281px;
   }
 }
+@supports (-webkit-touch-callout: none) {
+  .v2-header-3-text {
+    letter-spacing: -2px;
```
[the remark this answers is not in the document]
Comment (cp-priyanka-g) [reply vue-frontend/src/components/home-new/BlogSection.vue:275]: yes. It is valid and used to decrease the spacing.

File: vue-frontend/src/components/home-new/LandingSection.vue (hunk 1 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/home-new/LandingSection.vue:386]: changed.

File: vue-frontend/package.json [hunk 3]
```diff
@@ -32,7 +32,7 @@
     "cross-env": "^7.0.3",
     "eslint": "^8.7.0",
     "eslint-plugin-vue": "^8.5.0",
-    "express": "^4.17.3",
+    "express": "^4.18.1",
```
Comment (cp-shruti-s) [inline vue-frontend/package.json:35]: No need to push package.json file changes

**Pull request closed without merging**